Overviews/autho4api_overview.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - Authorisation Framework for Network APIs Overview</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="Scripts/specifications.js"></script>
```

Comment:
No need for this javascript unless you wanted the same functionality as the specifications page(s).

i.e. those that have an id="releases-table"